lib/utils.js

```diff
@@ -70,7 +70,12 @@ class SpawnError extends Error {
  */
 function spawnProcess(command, args, options) {
   return new BbPromise((resolve, reject) => {
-    const child = childProcess.spawn(command, args, options);
+    const child = childProcess.spawn(command, args, {
+      ...options,
```

Comment:
What if we move this line to the end, such that the new `shell` option works like an overridable default?

Reply:
I asked myself the same question, but we shouldn't be able to let the `shell` option overridable otherwise, we might have some issue on Windows (as we did now)

Reply:
That's fair. Given that this is kind of an internal API, we are should be safe to enforce this in case we need to handle more code paths in the future.